config/manager/controller-config.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: numaplane-controller-config
 data:
   config.yaml: |
-    syncTimeIntervalMs: 30000
+    logLevel: 1
     includedResources: "group=apps,kind=Deployment;\
     group=,kind=ConfigMap;group=,kind=Secret;group=,kind=ServiceAccount;\
     group=numaflow.numaproj.io,kind=;\
```

Comment:
@xdevxy I guess we shouldn't need `numaflow.numaproj.io` and `numaplane.numaproj.io` in here anymore, right?

Reply:
yeah, we can take out that. @shubhamdixit863 can you take that out and try if you still able to deploy numaflow controller through `NumaflowControllerRollout`